tests/eas/load_tracking.py

```diff
@@ -811,6 +811,74 @@ def test_util_task_migration(self, experiment, tasks):
         """
         return self._test_util_per_cpu(experiment, tasks)
 
+class TwoTasksCPUMigrationTest(_CPUMigrationBase):
+    """
+    Goal
+    ====
+    Checks that the utilization is correctly updated for CPU when two tasks are
+    migrated from a CPU to another.
+
+    Detailed Description
+    ====================
+    Three tasks are created. Each task defined two phases as follow (the CPU
+    is the one corresponding for the hikey960 it may be another CPU on
+    another platform):
+
+              Phase 0         Phase 1
+            CPU | Util      CPU | Util
+    Task 0:   4 | 20%         5 | 20%
+    Task 1:   5 | 50%         4 | 50%
+
+    Expected Behaviour
+    ==================
+    The utilization of a CPU is sampled during both phases and should be
+    observed as follow:
+
+            Phase 0         Phase 1
+            Util            Util
+    CPU 4:   20%             50%
+    CPU 5:   50%             20%
+    """
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Get the 2 CPUs with the highest capacities
+        cap_cpu = sorted(test_env.calibration(),
+                         key=test_env.calibration().get)[:2]
+        tasks = []
+        # tasks creation
+        task0_phase0 = Periodic(duty_cycle_pct=20, duration_s=1, period_ms=16,
+                                cpus=[cap_cpu[0]])
+        task0_phase1 = Periodic(duty_cycle_pct=20, duration_s=1, period_ms=16,
+                                cpus=[cap_cpu[1]])
+        tasks.append(task0_phase0 + task0_phase1)
+
+        task1_phase0 = Periodic(duty_cycle_pct=50, duration_s=1, period_ms=16,
+                               cpus=[cap_cpu[1]])
+        task1_phase1 = Periodic(duty_cycle_pct=50, duration_s=1, period_ms=16,
+                               cpus=[cap_cpu[0]])
+        tasks.append(task1_phase0 + task1_phase1)
+
+        json_file = cls.generate_json(test_env, tasks, 'CPUMigration1')
+        wload = cls.get_wload(json_file+'.json', len(tasks))
+        conf = {
+            'tag' : '2tasks_cpu_migrate',
```

Comment:
`cpu_migrate_two_tasks` ?

Reply:
Done (cpu_migration_two tasks for consistency with previous comment)